plugin/core/views.py

```diff
@@ -28,6 +28,7 @@
 import sublime
 import sublime_plugin
 import tempfile
+from ..semantic import SEMANTIC_SCOPES
```

Comment:
It's not really "allowed" to import non-core stuff from 'core'.
So it looks like it wasn't not a good idea to have `init_region_keys` here. Or `SEMANTIC_SCOPES` should maybe be a core thing.

Reply:
also initializing keys in this order doesn't really gives us desired behavior every time, seems like this only works 50% of the time,  `lsp_highlight_{}` keys somehow manages to get initialized before this ever gets called.
would you happen to know what causes that?